### PR TITLE
Help aws users find the configuration information they need

### PR DESCRIPTION
**Pull request by rromanchuk**
Related: 
https://github.com/toptal/chewy/pull/301

https://github.com/toptal/chewy/pull/302

### CODE REVIEW

**Comment by pyromaniac** [comment]
Hey man! Sorry I missed this somehow. Can you please resolve the conflict so I could merge it?

**Comment by pyromaniac** [comment]
Never mind, I did it, thanks for this contribution!

File: README.md [hunk 1]
```diff
@@ -15,6 +15,7 @@ Chewy is an ODM and wrapper for [the official Elasticsearch client](https://gith
 * [Why Chewy?](#why-chewy)
 * [Usage](#usage)
   * [Client settings](#client-settings)
+    * [AWS ElasticSearch configuration] (#aws-elastic-search)
```
Comment (rajn-webonise) [inline README.md]: Hey @pyromaniac, there's an extra white-space here, which is messing up the Markdown styling.
It should be:
`    * [AWS ElasticSearch configuration](#aws-elastic-search)`

Comment (pyromaniac) [inline README.md]: Will fix it in another PR, thanks.

**Comment by andrhamm** [comment]
There seems to be a dependency required for `aws_signers_v4`? Is that correct? It would be great if the documentation reflected the required dependencies and versions to use.

**Pull request merged**